html/internal_dashboard/js/Arr/CreateGroupTasksPanel.js

```diff
@@ -0,0 +1,1398 @@
+Ext.namespace('XDMoD', 'XDMoD.Arr');
+XDMoD.Arr.date_format = 'm/d/Y';
```

Comment:
We should us a consistent date format throughout XDMoD. The standard used elsewhere is Y-m-d.